BBB/Präs-Downloader.py

```diff
@@ -2,18 +2,29 @@
 import io
 from PyPDF2 import PdfFileMerger
 import urllib
+import tkinter as tk
+from tkinter import filedialog
+
+root = tk.Tk()
+root.withdraw()
+
 i = 1
 merger = PdfFileMerger()
-url = input("URL with the ending '/svg/{}': ")
-name = input("Name of the file: ")
+split = input("URL: ").split("/")
+split.pop()
+url = "/".join(split) + "/{}"
+name = filedialog.asksaveasfile(
```

Comment:
Name via input as before
```suggestion
name = intput("Name: ")
```